src/widgets/main/ui/main.tsx

```diff
@@ -10,7 +10,7 @@ export const Main = () => {
         <div className="left">
           <div className="title-container">
             <MainTitle>RS School</MainTitle>
-            <Subtitle text="Free courses. High motivation" />
+            <Subtitle>Free courses. High motivation</Subtitle>
```

Comment:
You need to use a .medium .black subtitle here

Reply:
We don't need pass that variants, because default is:

    defaultVariants: {
      size: 'medium',
      color: 'gray',
    },
 
  Plus, in site right now we have these styles: gray, not black. (18px font - is because media-tablet apply): 
  ![Screenshot 2024-07-11 at 22 33 41](https://github.com/rolling-scopes/site/assets/69956633/43f4272b-a6c5-4a82-aa16-ecdd1809645a)